CPPLINT.cfg

```diff
@@ -1,3 +1,3 @@
 set noparent
-filter=-runtime/reference
+filter=-runtime/reference,-runtime/string
```

Comment:
a quick glance did not make the necessity of this clear. I assume this is for your test?

Reply:
Yes, it's for the
```
static const std::string
```
members in KamikazeProcessor, but it's not the only place we would like to have it, see further references:
 https://github.com/apache/nifi-minifi-cpp/pull/670/files#diff-dfeecc634b3aea462f1f8dc52c2abaceR50

This rule forces people to use raw pointers instead of string, which is simply error-prone, but doesn't add value.